### ISSUE TITLE
Line Feeds in Firefox Browse Mode

### ISSUE THREAD

**Issue by dan1982code** [nvaccess/nvda#7346]
Open the attached in Firefox and read by word (ctrl+right/ctrl+left). NVDA announces "line feed" as a word at the end of each line. This is caused in the first case by a space at the end of line 1. Two line feeds are announced in a row after line 2 because of the trailing space on line 2 and the blank line.

I would expect these line feeds not to read when moving by word.
[vm.txt](https://github.com/nvaccess/nvda/files/1117562/vm.txt)

**Comment by Adriani90** [comment]
This is the case in all browsers as well as in all text editors. I tested in Firefox, Crhome, IE, Wordpad, Notepad and MS Word. @feerrenrut, @jcsteh I guess this is actually intended behabior but I do not have an argument at hand to explain why. What is the reason behind this?

**Comment by Adriani90** [comment]
Note, you don't have to change the file to html, just open the txt file in Firefox for this to reproduce.
Actually, I expect NVDA to report line feed or blank only for the blank line. All other line feeds characters should be suppressed and the word on the next line should be read instead.

**Comment by Adriani90** [comment]
Sorry, after testing further, i.e. creating a text in MS Word with multiple lines, the line feed character is read also. So this behavior is consistent every where when line feed characters are added. AFter thinking further, I think it is good to have these characters reported even when using word by word navigation because it gives you a picture of the text structure. If these characters are not announced, then you would not know when the line end and starts unless you change the navigation from word to character which could be inefficient.

So I am closing this issue as works for me, since this behavior is expected and it makes sense.
